### PR TITLE
Kubelet: minor cleanups

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

File: pkg/kubelet/dockertools/manager.go [hunk 1]
```diff
@@ -1067,9 +1067,8 @@ func (dm *DockerManager) killContainer(containerID types.UID) error {
 	return err
 }
 
-// TODO(vmarmol): Unexport this as it is no longer used externally.
 // Run a single container from a pod. Returns the docker container ID
-func (dm *DockerManager) RunContainer(pod *api.Pod, container *api.Container, netMode, ipcMode string) (kubeletTypes.DockerID, error) {
+func (dm *DockerManager) RunContainerInPod(pod *api.Pod, container *api.Container, netMode, ipcMode string) (kubeletTypes.DockerID, error) {
```
Comment (vmarmol) [inline pkg/kubelet/dockertools/manager.go]: Can we unexport this method?

Comment (yujuhong) [inline pkg/kubelet/dockertools/manager.go]: ..........That was my intention, if only I had remember to change R to r.

Comment (vmarmol) [inline pkg/kubelet/dockertools/manager.go]: :D

**Comment by vmarmol** [comment]
LGTM, will merge on green. Thanks @yujuhong!

**Pull request merged**